packages/dds/tree/src/test/tableSchema.spec.ts

```diff
@@ -25,75 +22,48 @@ import type {
 } from "../util/index.js";
 import { validateUsageError } from "./utils.js";
 import { takeJsonSnapshot, useSnapshotDirectory } from "./snapshots/index.js";
+// eslint-disable-next-line import/no-internal-modules
+import { describeHydration } from "./simple-tree/utils.js";
 
 const schemaFactory = new SchemaFactoryAlpha("test");
 
-describe("TableFactory unit tests", () => {
-	function createTableSchema() {
-		class Cell extends schemaFactory.object("table-cell", {
-			value: schemaFactory.string,
-		}) {}
-
-		class ColumnProps extends schemaFactory.object("table-column-props", {
-			/**
-			 * Label text for the column.
-			 */
-			label: schemaFactory.optional(schemaFactory.string),
-		}) {}
-		class Column extends TableSchema.column({
-			schemaFactory,
-			cell: Cell,
-			props: ColumnProps,
-		}) {}
-
-		class RowProps extends schemaFactory.object("table-row-props", {
-			/**
-			 * Whether or not the row is selectable.
-			 * @defaultValue `true`
-			 */
-			selectable: schemaFactory.optional(schemaFactory.boolean),
-		}) {}
-		class Row extends TableSchema.row({
-			schemaFactory,
-			cell: Cell,
-			props: schemaFactory.optional(RowProps),
-		}) {}
-
-		class Table extends TableSchema.table({
-			schemaFactory,
-			cell: Cell,
-			column: Column,
-			row: Row,
-		}) {}
-
-		return {
-			Cell,
-			Column,
-			Row,
-			Table,
-		};
-	}
+class Cell extends schemaFactory.object("table-cell", {
```

Comment:
Note for reviewers: extracts the common schema out of the fixture. Tests that use custom schema now name theirs "MyFoo" to avoid conflicting. This simplifies things quite a bit.